components/service/glean/src/test/java/mozilla/components/service/glean/private/EventMetricTypeTest.kt

```diff
@@ -197,11 +202,14 @@ class EventMetricTypeTest {
         Glean.setUploadEnabled(false)
         assertEquals(false, Glean.getUploadEnabled())
         eventMetric.record(mapOf(testNameKeys.testName to "event2"))
-        val snapshot2 = eventMetric.testGetValue()
-        assertEquals(1, snapshot2.size)
+        try {
```

Comment:
Let's use the assertion with exception [utility](https://kotlinlang.org/api/latest/kotlin.test/kotlin.test/assert-fails-with.html) here.

Reply:
Good catch.  Done.

Reply:
That library doesn't seem to be available to us in `a-c`, unfortunately.

Reply:
Ouch, no problem then!